Working group "About" tab description and routing

Ilichhh opened the pull request:
- [x] Add default description for WG if metadata is not set by WG Lead
- [x] Fix broken Handbook links
- [x] Add routing to WG page

[The working "About" tab is empty (for all working groups) #4654](https://github.com/Joystream/pioneer/issues/4654)

thesan reviewed (approved):
My suggestion [here](https://github.com/Joystream/pioneer/pull/4771#discussion_r1487607750) was wrong since the `WorkingGroup` component is now actually the About tab. What you implemented instead works but it duplicates a lot of code for testing just one page.

Instead I replaced `WorkingGroup` by `WorkingGroupsModule` which works more like the original suggestion. But I wanted to make sure it worked first so I committed it myself.

The rest LGTM !

Ilichhh commented:
> My suggestion [here](https://github.com/Joystream/pioneer/pull/4771#discussion_r1487607750) was wrong since the `WorkingGroup` component is now actually the About tab. What you implemented instead works but it duplicates a lot of code for testing just one page.
> 
> Instead I replaced `WorkingGroup` by `WorkingGroupsModule` which works more like the original suggestion. But I wanted to make sure it worked first so I committed it myself.
> 
> The rest LGTM !

Cool, thanks!  
But you just removed some `gql.queries` that I added, and wornings on `History` and `Openings` tabs appears again. I don't think they're really needed since they contain empty data. Plus other stories also have a lot of missing gueries and there are wornings. Just want to clarify this point for better understanding for the future.
